receiver/netflowreceiver/producer.go

```diff
@@ -45,11 +47,19 @@ func (o *OtelLogsProducerWrapper) Produce(msg any, args *producer.ProduceArgs) (
 	logRecords := scopeLog.LogRecords()
 
 	// A single netflow packet can contain multiple flow messages
-	for _, msg := range flowMessageSet {
-		logRecord := logRecords.AppendEmpty()
-		parseErr := addMessageAttributes(msg, &logRecord)
-		if parseErr != nil {
-			continue
+	if o.sendRaw {
+		for _, msg := range flowMessageSet {
+			logRecord := logRecords.AppendEmpty()
+			logRecord.Body().SetStr(fmt.Sprintf("%+v", msg))
+		}
+	} else {
+		// Parse the message and add the attributes to the log record
+		for _, msg := range flowMessageSet {
+			logRecord := logRecords.AppendEmpty()
+			parseErr := addMessageAttributes(msg, &logRecord)
+			if parseErr != nil {
+				continue
+			}
 		}
 	}
```

Comment:
Wouldn't this be more readable?
```suggestion
	for _, msg := range flowMessageSet {
		logRecord := logRecords.AppendEmpty()
		if o.sendRaw {
			logRecord.Body().SetStr(fmt.Sprintf("%+v", msg))
		} else {
			// Parse the message and add the attributes to the log record
			_ := addMessageAttributes(msg, &logRecord)
		}
	}
```

Note that I've removed the `if parseErr != nil { continue }` because it's at the end of the loop, so it has no effect, right?